### ISSUE TITLE
Asynchronous Update to a page 

### ISSUE THREAD

**Issue by Brianzhengca** [ChrisKnott/Eel#89]
Sorry for being a noob at python but is there any way to asynchronously update page using eel? I am trying to make a calculator program with eel and need the result to update asynchronously. So far I haven't found any tutorial on this issue. Any help will be appreciated. Thank you.

**Comment by brumar** [comment]
This Can you share

**Comment by brumar** [comment]
This should be easy. Can you share what you have tried so far ? As communications are bi-directionnal you have two possible path. In any case an exposed python function doing the computation will be called by javascript ... and javascript will update the page. Either you take the return value of the python function or your python function will call a third javacript function dedicated to the update of the page. In any case, every aspect directly related to the frontend (getting the event, update the page) is under the responsability of javascript.

**Comment by Brianzhengca** [comment]
@brumar I haven't really started with anything yet outside of an rough idea about the basic structures of my application. I just came across [this tutorial](https://nitratine.net/blog/post/python-gui-using-chrome/) however, in that tutorial it only mentioned a little about asynchronous communication between python and javascript and did not mention how to asynchronously update a page. In the past I have always used tkinter for my gui applications but I decided to switch because I feel like tkinter looks ugly. Thank you
